docker/Dockerfile.profile

```diff
@@ -0,0 +1,18 @@
+# base image that creates all necessary dependencies for
```

Comment:
This file can be removed entirely.  Its changes have already been incorporated in to the main Dockerfiles.